extensions/amp-carousel/0.2/test-e2e/test-carousel.js

```diff
@@ -21,32 +21,27 @@ import {
 } from './helpers';
 
 describes.endtoend('AMP carousel', {
+  fixture: 'http://localhost:8000/test/manual/amp-carousel-0-2/basic.amp.html',
```

Comment:
One of the nice things about `describes.integration` is that it abstracts away boilerplate markup (e.g. `<head>` content) and inlines the test fixture. This also makes it easy to reuse a fixture/test tuple in different validator specs. Check out #19943.

I think we should also do that here if possible.